app/db/postgres.py

```
@@ -0,0 +1,13 @@
+import psycopg2
+from psycopg2.extras import RealDictCursor
+from app.config.settings import settings
+
+def get_postgres_connection():
+    try:
+        conn = psycopg2.connect(settings.POSTGRES_URL, cursor_factory=RealDictCursor)
+        print("PostgreSQL conectado correctamente")
+        return conn
```

Ruff (0.13.3)

Consider moving this statement to an `else` block

(TRY300)